.gitignore

```diff
@@ -1,6 +1,8 @@
 # Ignore generated files
 # ...
 
+.gitignore
```

Comment:
I think it shouldn't be here...

Reply:
Can you remove it, or do you need i do another PR with this file removed ?

Reply:
send another PR please, also revert the Makefile.Android rename...

Reply:
well i tried to do another PR, but git is not my cup of tea. Feel free to grab Makefile.Android.linux directly, if you want.